rest_api/src/test/java/tqs/cloudit/services/StepsDefs.java

```diff
@@ -247,17 +249,17 @@ public void badRegister() {
     public void checkErrorRegisterMessage() {
         new WebDriverWait(driver,10L).until((ExpectedCondition<Boolean>) 
                 (WebDriver d) -> { 
-                                        try 
-                                        { 
-                                            driver.switchTo().alert(); 
-                                            driver.switchTo().alert().accept();
-                                            return true; 
-                                        }   // try 
-                                        catch (NoAlertPresentException Ex) 
-                                        { 
-                                            return false; 
-                                        }   // catch 
-                                    });
+                            try 
```

Comment:
This code is no compliant with the code style, careful with indentation and brackets position!